modules/metering-uninstall.adoc

```diff
@@ -3,25 +3,60 @@
 // * metering/metering-uninstall.adoc
 
 [id="metering-uninstall_{context}"]
-= Uninstalling metering from {product-title}
+= Uninstalling the Metering Operator from {product-title}
 
-You can remove metering from your cluster.
+The following guide provides details on removing the Metering Operator and its components.
 
-.Prerequisites
+[NOTE]
+====
+Metering does not manage or delete any S3 bucket data. If you used Amazon S3 for storage, any S3 buckets used to store metering data must be manually cleaned up.
+====
+
+== Removing the Metering Operator
+
+You can remove the Metering Operator by following the documentation on xref:../../operators/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-a-cluster[deleting Operators from a cluster].
+
+== Uninstalling metering
+
+To uninstall metering, you must remove its custom resource and delete the `openshift-metering` namespace.
 
-* Metering must be installed.
+. Prerequisites
 
-.Procedure
+. You have uninstalled Metering following the documentation listed in the previous step.
 
-To remove metering:
+. Procedure
 
-. In the {product-title} web console, click *Operators* -> *Installed Operators*.
+. Use the following command to remove all resources created by the Metering Operator:
++
+----
+$ oc --namespace openshift-metering delete meteringconfigs.metering.openshift.io openshift-metering
+----
 
-. Find the Metering Operator and click the {kebab} menu. Select *Uninstall Operator*.
+. Once that command has completed, verify all of the pods listed in the `openshift-metering` namespace have been deleted, and run the following command:
++
+----
+$ oc delete namespace openshift-metering
+----
 
-. In the dialog box, click *Remove* to uninstall metering.
+== Uninstalling metering CRDs
+
+Once the Metering Operator has been uninstalled and the `openshift-metering` namespace has been deleted, the metering CRDs will still exist in the cluster.
 
 [NOTE]
 ====
-Metering does not manage or delete any S3 bucket data. If you used Amazon S3 for storage, any S3 buckets used to store metering data must be manually cleaned up.
+Deleting the metering CRDs will disrupt any additional metering installations. Ensure no other installations exist before proceeding.
 ====
+
+. Prerequisites
+
+. You have deleted the `MeteringConfig` custom resource.
+
+. You have deleted the `openshift-metering` namespace.
+
+. Procedure
+
+. Delete the remaining metering CRDs be running the following command:
++
+----
+$ oc get crd -o name | grep "metering.openshift.io" | xargs oc delete
```

Comment:
I, unfortunately, don't have a way of cleanly deleting these resources without using tooling like `xargs`.